### ISSUE TITLE
Asking to be unbanned.

### ISSUE THREAD

**Issue by Revolution641** [sm64js/sm64js#777]
Hello. Apologies in advanced if this happens to be wasting your time, or this is just not the right area for a unban request.
A few years ago in 64 JS, I pretty much got my mental health screwed up, and ended up saying...a lot of problematic comments.
Since then, I've gotten older, and have been able to control my mental health better. I have gotten exceptionally along with plenty of communities too.
I was known simply as "mgaming99" (Yes, pathetic now that I think back.)
I am requesting for a possible unban from the game, as I wish to be able to play with friends.
If any details are needed, I suggest contacting me on Discord:
Rev64#2063

**Comment by Revolution641** [comment]
Oh, I forgot to mention. I was accused of being underaged too. I am currently 15 years old, and in a month will be 16.

**Comment by drok28** [comment]
Hey, I've done this exact same thing a few times. I feel you, Rev.

**Comment by Revolution641** [comment]
They aren't gonna do anything are they.

**Comment by drok28** [comment]
Most likely not, but there are other Mario multiplayer games on a browser.

**Comment by EveryDayThrowWay** [comment]
> Most likely not, but there are other Mario multiplayer games on a browser.

I've been looking for more to play what are some other ones :0

**Comment by drok28** [comment]
> > Most likely not, but there are other Mario multiplayer games on a browser.
> 
> I've been looking for more to play what are some other ones :0

Let's see, there's New Super Mario Bros. Versus, and there's Mario Royale.

**Comment by Blu3killa25** [comment]
hello i have been banned, could you please unban me, my ip is banned because my friend would not stop spamming the n-word. if you could just ban him, or unban the whole ip that would be great.
Blu3killa 25
Thx

**Comment by drok28** [comment]
> hello i have been banned, could you please unban me, my ip is banned because my friend would not stop spamming the n-word. if you could just ban him, or unban the whole ip that would be great. Blu3killa 25 Thx

I feel like the friend should be banned because he was saying the n-word, not you.

**Comment by Blu3killa25** [comment]
> 

yes i feel the same but since we are playing on the same ip i was banned to

**Comment by drok28** [comment]
> > 
> 
> yes i feel the same but since we are playing on the same ip i was banned to

Totally not fair.

**Comment by Someone567123** [comment]
if you guys wanna do stuff like that then just play sm64ex co-op. as far as
i know, all chats in the game are private, unless someone leaks them.

sm64ex co-op is another project Agent X and other people are working on,
and i think it's way better than a browser-based mario game. if you don't
have a pc of your own (which i doubt since most sm64js players play on
their school laptops), then you can't really play it. It's available for
Windows, Linux, and Android, but you have to compile it yourself. did i
mention that you can use mods? how fun!

the way you play this game online is by authorizing the sm64ex co-op
application on discord, host the game, and send a game invite to the people
you desire to play with, or you can host it similar to how you would host a
minecraft server, but since discord invites are back, that's obsolete, but
still doable.

yeah it sucks that you guys got banned boohoo. you guys should have been
more careful or something.

**Comment by ImaginaryNeon** [comment]
> if you guys wanna do stuff like that then just play sm64ex co-op. as far as i know, all chats in the game are private, unless someone leaks them. sm64ex co-op is another project Agent X and other people are working on, and i think it's way better than a browser-based mario game. if you don't have a pc of your own (which i doubt since most sm64js players play on their school laptops), then you can't really play it. It's available for Windows, Linux, and Android, but you have to compile it yourself. did i mention that you can use mods? how fun! the way you play this game online is by authorizing the sm64ex co-op application on discord, host the game, and send a game invite to the people you desire to play with, or you can host it similar to how you would host a minecraft server, but since discord invites are back, that's obsolete, but still doable. yeah it sucks that you guys got banned boohoo. you guys should have been more careful or something.

I think the reason people tend to do it on their shool laptops is because they don't have many options for stuff to do while at shool.

**Comment by Someone567123** [comment]
true

**Comment by ImaginaryNeon** [comment]
I'd assume that given your case, there's likely still hope to clear your name. Maybe try the Discord, but for me I doubt that would work.
